src/style.css

```diff
@@ -1,3 +1,95 @@
+html {
+  font-family: Roboto, Arial, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  font-style: normal;
+  line-height: 14px;
+}
+
+* {
+  box-sizing: border-box;
```

Comment:
Avoid using the `*` selector as it impacts performance. It's better to set styles only for elements that require them.